pitch.md

```diff
@@ -23,38 +58,137 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+I don't know my exact syntax. But for the sake of simplicity, let us focus on 
+"full circle" moves that do not touch the body, shorten string length, or 
+involve string manipulation.
+
+There are two main properties that distinguish one move from another:
+
+1. Where the glowsticks are in relation to the body
+	* Plane
+		1. Wall - perpendicular to front or back of body
+		2. Wheel - perpendicular to left or right of body
+		3. Floor - perpendicular to the floor
+	* Is the arm rotating (moving in circles), and if so, counterclockwise or clockwise?
+	* Is the glowstick moving counterclockwise or clockwise relative to the hand?
+	* Start and end position of the hand after one full 360° circle.
+2. Where the glowsticks are in relation to each other
+	* This can be represented as offset angle phases, as many moves have both glowsticks doing the same motion but starting at different phase positions.
+
+Ideally, with this information, we can build individual tricks based on these 
+properties for a "full circle".
+
```

Comment:
It seems like this would be best implemented as an external DSL to allow people to easily describe the features of each move. I think you have a good outline for what you want the users to be able to express in your DSL, and it'll just be a matter of designing a concise yet readable syntax!